CMakeLists.txt

```diff
@@ -450,10 +450,23 @@ if(PKG_CONFIG_FOUND)
 else()
 	find_package(LIBUNWIND QUIET)
 endif()
+
 if(NOT LIBUNWIND_FOUND)
 	message(WARNING "libunwind not found. Stack traces from tests will not be reliable")
 endif()
 
+if(PKG_CONFIG_FOUND)
+	pkg_check_modules(PYTHON3 QUIET python3)
+else()
+	find_package(PYTHON3 QUIET)
```

Comment:
https://stackoverflow.com/questions/29245558/how-to-link-with-python3-libs-with-cmake
cmake comes with a module for findin python, so I guess these five lines could be replace by something like:
`find_package(PythonLibs 3 REQUIRED)`

Or this one to just find the interpreter:
https://cmake.org/cmake/help/v3.0/module/FindPythonInterp.html

Reply:
So let's try ```find_package(PythonInterp 3 QUIET REQUIRED)``` ...

Reply:
@GBuella please resolve